packages/stark-ui/assets/themes/_theming.scss

```diff
@@ -0,0 +1,49 @@
+@import "~@angular/material/theming";
+@import url("https://fonts.googleapis.com/css?family=Material+Icons");
+@import url("https://fonts.googleapis.com/css?family=Roboto:300,400,500,700");
```

Comment:
As we talked with Christopher, in waiting of a choice for fonts, we should simply use the default font of the browser itself instead of downloading one from google.